test/tests_cuda.jl

```diff
@@ -43,8 +43,8 @@ end
 	y = round.(rand(Float32, N) * 100)
 	alpha = 2.5
 
-	x_device = JACC.Array(x)
-	y_device = JACC.Array(y)
+	x_device = JACC.array(x)
```

Comment:
Currently I only updated/tested the CUDA backed. But as of now all unittests pass for CUDA and `threads` backends.